src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -204,7 +204,9 @@ private void validateSchema(final EventType eventType) throws InvalidEventTypeEx
         try {
             final JSONObject schemaAsJson = new JSONObject(eventType.getSchema().getSchema());
 
-            if (hasReservedField(eventType, schemaAsJson, "metadata")) {
+            if (eventType.getCategory() == EventCategory.BUSINESS
```

Comment:
Why you now check it only for BUSINESS event type? Is that because for DATA event schema checks only "data" field?